src/lib/modules/blockchain_process/blockchain.js

```diff
@@ -63,32 +64,42 @@ var Blockchain = function(userConfig, clientClass) {
     proxy: this.userConfig.proxy
   };
 
-  if (this.userConfig === {} || this.userConfig.default || JSON.stringify(this.userConfig) === '{"enabled":true}') {
-    this.config.account = {};
+  if (this.userConfig.accounts) {
+    const nodeAccounts = this.userConfig.accounts.find(account => account.nodeAccounts);
+    if (nodeAccounts) {
+      this.config.account = {
+        numAccounts: nodeAccounts.numAddresses || 1,
+        password: nodeAccounts.password,
+        balance: nodeAccounts.balance
+      };
+    }
+  }
+
+  if (this.userConfig === {} || this.userConfig.default || JSON.stringify(this.userConfig) === '{"ethereumClientName":"geth"}') {
```

Comment:
```
JSON.stringify(this.userConfig) === '{"ethereumClientName":"geth"}'
```

This looks very weird to me. Any reason we don't check for 

```
userConfig.ethereumClientName && userConfig.ethereumClientName === 'geth'
```

Reply:
From what I understand, stringify removes all the falsy values, so we end up with just `'{"ethereumClientName":"geth"}'`. Only testing for the `ethereumClientName` would be bad, because we would override to the defaults pretty much all the time, whereas this checks that we used no configs.
Anyway, I agree it's weird. That use case it pretty much only used by the unit tests from what I can see.